lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -85,8 +85,13 @@ def test_as_inactive_user(self):
         self.verify_response(params={'username': inactive_user.username})
 
     def test_missing_username(self):
+        response_to_anonymous = self.verify_response(expected_status_code=200)
         self.setup_user(self.honor_user)
-        self.verify_response(expected_status_code=400)
+        response_to_missing_username = self.verify_response(expected_status_code=200)
+        self.assertEqual(response_to_anonymous.data, response_to_missing_username.data, "Response to anonymous user did"
+                                                                                        " not match response to "
```

Comment:
Nit: Please format according to style guide - with each argument in its own line.